app/services/notification_service.rb

```diff
@@ -62,6 +67,13 @@ def target_match_request_params(player_ids)
     }
   end
 
+  def msg_request_params(player_ids, text)
+    { 'app_id' => ENV['ONE_SIGNAL_APP_ID'],
```

Comment:
I would move this to a constant above
`ONE_SIGNAL_APP_ID = ENV['ONE_SIGNAL_APP_ID']`